pyiode/iode_database/variables_database.pyx

```diff
@@ -735,6 +742,8 @@ cdef class Variables(_AbstractDatabase):
         1980Y1:2010Y1
 
         >>> # update sample by passing a slice
+        # GDM> AFAIK, this is (unfortunately) not a valid syntax in Python?!? (at least Python <= 3.11)
```

Comment:
The doctests are automatically runned for Python 3.9, 3.10 and 3.11 on Github and this syntax actually works.

Reply:
Are you REALLY sure this particular doctest is run?!?
I am really 99.99% sure it is not valid... and my quick test with an online Python (don't have Python on this computer) seems to confirm my memory:

```python
>>> class A:
...     def __setattr__(self, key, value):
...         print("setattr", key, value)
>>> a = A()
>>> a.variables = 'a':'b'
```
yields
```python
    a.variables = 'a':'b'
                     ^
SyntaxError: invalid syntax
```

Reply:
Yes, you're right. I didn't noticed that doctests from the setters are not runned. Thanks for noticing it.

Reply:
docstring of setters have been merged into docstring of getters -> see PR #573